modules/features2d/misc/java/test/BRIEFDescriptorExtractorTest.java

```diff
@@ -28,7 +27,7 @@ private Mat getTestImg() {
     @Override
     protected void setUp() throws Exception {
         super.setUp();
-        extractor = DescriptorExtractor.create(DescriptorExtractor.BRIEF);
+        extractor = createAlgorythmInstance(XFEATURES2D_PACKAGE_NAME+"BriefDescriptorExtractor", ALGORITHM_FACTORY_NAME, null, null);
```

Comment:
I decided to drop "Algorythm" as well and call it "createClassInstance" as this method is  generic and can actually instantiate any class both using "new" operator and factory method technique. I thought it may be useful for other tests as well.